### PR TITLE
Wallet: Add fromMnemonic function

### PR DESCRIPTION
**Pull request by kaliubuntu0206**
Backporting https://github.com/ethereumjs/ethereumjs-wallet/pull/192

### CODE REVIEW

File: packages/wallet/src/hdkey.ts [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
    */
   public static fromMasterSeed(seedBuffer: Uint8Array): EthereumHDKey {
     return new EthereumHDKey(HDKey.fromMasterSeed(seedBuffer))
   }
 
+  /**
+   * Creates an instance based on BIP39 mnemonic phrases
+   */
+  public static fromMnemonic(mnemonic: string, passphrase?: string): EthereumHDKey {
+    return EthereumHDKey.fromMasterSeed(Buffer.from(mnemonicToSeedSync(mnemonic, passphrase)))
```
Comment (paulmillr) [inline packages/wallet/src/hdkey.ts:17]: Remove `Buffer.from`, it should always consume Uint8Array, which it already returns

Comment (kaliubuntu0206) [reply packages/wallet/src/hdkey.ts]: Yes, just did it. Thank you.

**Comment by kaliubuntu0206** [comment]
Also just fixed the broken imports

**Review by paulmillr — DISMISSED** [review]

**Review by acolytec3 — APPROVED** [review]
LGTM. Thanks for the PR!

**Pull request merged**